otter/test/convergence/test_steps.py

```diff
@@ -1177,3 +1188,125 @@ def test_returns_retry(self):
         self.assertEqual(
             sync_perform(base_dispatcher, eff),
             (StepResult.RETRY, ['building']))
+
+
+class AppendStackUUIDTests(SynchronousTestCase):
+    """Tests for :func:`append_stack_uuid`."""
+
+    def test_normal_use(self):
+        """Ensures correct behavior."""
```

Comment:
This docstring is a bit pointless ;) How about "The stack ID is inserted into the given stack configuration" or something.